### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by linusromland**

### CODE REVIEW

**Comment by linusromland** [comment]
Merge @superfroggman

**Comment by superfroggman** [comment]
Much nicer readME but I think that you should add default port for added clarity @linusromland

**Comment by linusromland** [comment]
@superfroggman fixed your thing

**Comment by linusromland** [comment]
cool

**Comment by linusromland** [comment]
cool merge

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CPU Socket
 Graph showing CPU, Memory and Disk usage on the server. <br/>
+
+The default port is 1234
```
Comment (superfroggman) [inline README.md:4]: nice

**Pull request merged**